typescript/utils/src/objects.test.ts

```diff
@@ -13,4 +13,37 @@ describe('Object utilities', () => {
     expect(deepCopy({ a: 1, b: 2 })).to.eql({ a: 1, b: 2 });
     expect(deepCopy({ a: 1, b: 2 })).to.not.eql({ a: 1, b: 3 });
   });
+
+  it('objMerge', () => {
+    const obj1 = { a: 1, b: 2, c: { d: '4' } };
+    const obj2 = { b: 3, c: { d: '5' } };
+    const merged = objMerge(obj1, obj2);
+    expect(merged).to.eql({ a: 1, b: 3, c: { d: '5' } });
+  });
+
+  it('objMerge with array', () => {
```

Comment:
for completeness would also add a test to check arrays gets replaced if `mergeArrays` is `false`